src/ui/views/LandingPages/Welcome/AddressImport/SetPassword.tsx

```diff
@@ -0,0 +1,240 @@
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const [isMatch, setMatch] = useState(false);
+
+  const loadTempPassword = useCallback(async () => {
+    setPassword(tempPassword);
+  }, [tempPassword]);
+
+  useEffect(() => {
+    loadTempPassword();
+  }, [loadTempPassword]);
+
+  const successInfo = (message) => (
+    <Box
+      sx={{
+        width: '95%',
+        backgroundColor: 'success.light',
+        mx: 'auto',
+        borderRadius: '0 0 12px 12px',
+        display: 'flex',
+        flexDirection: 'row',
+        alignItems: 'center',
+      }}
+    >
+      <CheckCircleIcon size={24} color={'#41CC5D'} style={{ margin: '8px' }} />
+      <Typography variant="body1" color="success.main">
+        {message}
+      </Typography>
+    </Box>
+  );
+
+  const errorInfo = (message) => (
+    <Box
+      sx={{
+        width: '95%',
+        backgroundColor: 'error.light',
+        mx: 'auto',
+        borderRadius: '0 0 12px 12px',
+        display: 'flex',
+        flexDirection: 'row',
+        alignItems: 'center',
+      }}
+    >
+      <CancelIcon size={24} color={'#E54040'} style={{ margin: '8px' }} />
+      <Typography variant="body1" color="error.main">
+        {message}
+      </Typography>
+    </Box>
+  );
+
+  const [helperText, setHelperText] = useState(<div />);
+  const [helperMatch, setHelperMatch] = useState(<div />);
+
+  const handleImport = async () => {
+    setLoading(true);
+    try {
+      const accountKeyStruct = {
+        public_key: accounts[0].pubK,
+        sign_algo: getSignAlgo(accounts[0].signAlgo),
+        hash_algo: getHashAlgo(accounts[0].hashAlgo),
+        weight: 1000,
+      };
+      const result = await wallet.openapi.getLocation();
+      const installationId = await wallet.openapi.getInstallationId();
+      const userlocation = result.data;
+      const device_info = {
+        city: userlocation.city,
+        continent: userlocation.country,
+        continentCode: userlocation.countryCode,
+        country: userlocation.country,
+        countryCode: userlocation.countryCode,
+        currency: userlocation.countryCode,
+        device_id: installationId,
+        district: '',
+        ip: userlocation.query,
+        isp: userlocation.as,
+        lat: userlocation.lat,
+        lon: userlocation.lon,
+        name: 'FRW Chrome Extension',
+        org: userlocation.org,
+        regionName: userlocation.regionName,
+        type: '2',
+        user_agent: 'Chrome',
+        zip: userlocation.zip,
+      };
+      const address = accounts[0].address.replace(/^0x/, '');
+
+      await wallet.openapi.importKey(accountKeyStruct, device_info, username, {}, address);
+      await wallet.boot(password);
+      storage.remove('premnemonic');
+      await saveIndex(username);
+
+      if (pk) {
+        await wallet.importPrivateKey(pk);
+        goEnd();
+      } else {
+        await wallet.createKeyringWithMnemonics(mnemonic);
+        handleClick();
+      }
```

Comment:
This is really confusing what goEnd and handleClick do.

Reply:
I can change the handleClick to nextstep or gonext, it should go to the next tab

Reply:
That would be good. It's just hard to know what it does